apps/utils/locust_tests/v2/partially_adjudicated_post.py

```diff
@@ -0,0 +1,124 @@
+
+    The MBI list is randomly shuffled to get better sampling in sequential testing.
+    """
+
+    SERVICE_DATE = {"service-date": "gt2020-01-05"}
+    LAST_UPDATED = {"_lastUpdated": "gt2020-05-05"}
+    SERVICE_DATE_LAST_UPDATED = dict(SERVICE_DATE, **LAST_UPDATED)
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.mbis = list(master_pac_mbis)
+
+    def __post(self, resource, name, body=None):
+        params = {}
+        body = {} if body is None else body
+        body["mbi"] = random.choice(self.mbis)
+
+        self.run_task_by_parameters(
+            base_path=f"/v2/fhir/{resource}", params=params, body=body, name=name
+        )
+
+    @tag("claim")
+    @task
+    def get_claim(self):
+        """Get single Claim"""
+        self.__post("Claim/_search", "claim")
+
+    @tag("claim", "service-date")
+    @task
+    def get_claim_with_service_date(self):
+        """Get single Claim with service date"""
+        self.__post("Claim/_search", "claimServiceDate", self.SERVICE_DATE)
+
+    @tag("claim", "last-updated")
+    @task
+    def get_claim_with_last_updated(self):
+        """Get single Claim with last updated"""
+        self.__post("Claim/_search", "claimLastUpdated", self.LAST_UPDATED)
+
+    @tag("claim", "service-date", "last-updated")
+    @task
+    def get_claim_with_service_date_and_last_updated(self):
+        """Get single Claim with last updated and service date"""
+        self.__post("Claim/_search", "claimServiceDateLastUpdated", self.SERVICE_DATE_LAST_UPDATED)
+
+    @tag("claim", "exclude-samsa")
+    @task
+    def get_claim_with_exclude_samsa(self):
+        self.__post("Claim/_search", "excludeSAMSA", True)
+
+    @tag("claim-response")
+    @task
+    def get_claim_response(self):
+        """Get single ClaimResponse"""
+        self.__post("ClaimResponse/_search", "claimResponse")
+
+    @tag("claim-response", "service-date")
+    @task
+    def get_claim_response_with_service_date(self):
+        """Get single ClaimResponse with service date"""
+        self.__post("ClaimResponse/_search", "claimResponseServiceDate", self.SERVICE_DATE)
+
+    @tag("claim-response", "last-updated")
+    @task
+    def get_claim_response_with_last_updated(self):
+        """Get single ClaimResponse with last updated"""
+        self.__post("ClaimResponse/_search", "claimResponseLastUpdated", self.LAST_UPDATED)
+
+    @tag("claim-response", "service-date", "last-updated")
+    @task
+    def get_claim_response_with_service_date_and_last_updated(self):
+        """Get single ClaimResponse with last updated and service date"""
+        self.__post(
+            "ClaimResponse/_search",
+            "claimResponseServiceDateLastUpdated",
+            self.SERVICE_DATE_LAST_UPDATED,
+        )
+
+    @tag("claim", "exclude-samsa")
+    @task
+    def get_claim_response_with_exclude_samsa(self):
+        self.__post("ClaimRespons/_search", "excludeSAMSA", True)
```

Comment:
Typo here ^

Reply:
I would expect these tests to be failing with the wrong endpoint name here. Is that not the case?